src/pymodaq/extensions/pid/pid_controller.py

```diff
@@ -153,11 +153,11 @@ def ini_PID(self):
 
         else:
             if hasattr(self, 'PIDThread'):
-                if self.PIDThread.isRunning():
-                    try:
-                        self.PIDThread.quit()
-                    except Exception:
-                        pass
+                self.PIDThread.quit()
```

Comment:
same as above! it would also be easier to debug if this causes issues at the end..